<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project_Information</class>
 <widget class="QWidget" name="Project_Information">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informations</string>
  </property>
  <property name="locale">
   <locale language="French" country="Morocco"/>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>New OpenMC project </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Workspace</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4" colspan="2">
       <widget class="QPushButton" name="pBCreateNew">
        <property name="text">
         <string>Create new project</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Case</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="4">
       <widget class="QLineEdit" name="description"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Writen By</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QLineEdit" name="projectName"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="4">
       <widget class="QDateTimeEdit" name="dateTimeCreateNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string notr="true"/>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>12</hour>
          <minute>57</minute>
          <second>13</second>
          <year>2020</year>
          <month>10</month>
          <day>14</day>
         </datetime>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>d‏/M‏/yyyy HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="4">
       <widget class="QLineEdit" name="writtenBy"/>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="workspace"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Date Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="4">
       <widget class="QLineEdit" name="projectCase"/>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectName</tabstop>
  <tabstop>workspace</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>description</tabstop>
  <tabstop>projectCase</tabstop>
  <tabstop>writtenBy</tabstop>
  <tabstop>dateTimeCreateNew</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pBCreateNew</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
